crates/sui-e2e-tests/tests/unwrapped_then_deleted_tests.rs

```diff
@@ -13,86 +13,6 @@ mod sim_only_tests {
     use sui_types::base_types::ObjectID;
     use test_cluster::{TestCluster, TestClusterBuilder};
 
-    // This test exercise the protocol upgrade where we flip the feature flag
-    // simplified_unwrap_then_delete. It demonstrates the behavior difference before and after
-    // this upgrade in unwrapped_then_deleted and modified_at_versions fields in effects.
-    #[sim_test]
-    async fn test_simplified_unwrap_then_delete_protocol_upgrade() {
```

Comment:
Were these moved somewhere?

Reply:
This was deleted as it was causing issues (and was no longer needed). I'll ping out-of-band on it :)